source/manual/buckups.html.md

```diff
@@ -0,0 +1,123 @@
+---
+owner_slack: "#govuk-developers"
+title: Use of backups
+parent: "/manual.html"
+layout: manual_layout
+section: Backups
+type: learn
+---
+
+This manual provides a shared definition of a backup and documents our current use of backups summarising strategies and tools we use.
+
+> A backup is a process of creating a copy of the data stored in a database and storing it in a safe location to allow data recovery in case of a disaster, system failure or human error.
+
+# AWS RDS Backups
+
+## Point-in-time RDS backups
+
+Point-in-time backups for Amazon Relational Database Service (RDS) allow us to restore databases to a specific point in time in the past. This feature is useful when we want to restore database to a state before an unintended change or data loss occurred.
+
+When point-in-time backups for an RDS instance are enabled, Amazon RDS automatically takes regular backups of the database and stores them in Amazon S3. Each backup contains a snapshot of the database at a specific point in time. We can restore the database to any of these points in time, within the retention period.
```

Comment:
I think this is nearly right. I think RDS takes regular snapshots, but also stores database transaction logs in S3 at 5 minute intervals. For a restore to any point in time (within the backup retention period), it'll restore the snapshot before the desired time, and then "replay" the transaction logs on top of that state until the exact time required.

https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_PIT.html

Reply:
Thanks, added that info to the paragraph.